lib/python_version.sh

```diff
@@ -432,7 +432,23 @@ function python_version::warn_if_python_version_file_missing() {
 
 	case "${python_version_origin}" in
 		default | cached)
-			# TODO: Add a warning for this case.
+			output::warning <<-EOF
+				Warning: No Python version was specified.
+
+				Your app doesn't specify a Python version and so the buildpack
+				picked a default version for you.
+
+				Relying on this default version isn't recommended, since it
+				can change over time and may not be consistent with your local
+				development environment, CI or other instances of your app.
+
+				Please configure an explicit Python version for your app.
+
+				${instructions}
```

Comment:
Only recommendation would be adding some git instructions like https://github.com/heroku/heroku-buildpack-ruby/blob/f506e9ae3931bbce010fbcaa37c59fd26638f3c0/lib/language_pack/helpers/bundler_wrapper.rb#L225-L226

Otherwise, they might have the file locally, but forget to check it in.

Reply:
Ah great idea - thank you!

Reply:
@schneems Ah great idea - thank you!

Reply:
Added in 2211bb8d4bb6404aa795ba346579eefa288c606e